Articles/Blog/PracticesPerformancePortableEco.md

```diff
@@ -0,0 +1,75 @@
+# Practices for Productive Software Development using a Stable Performance Portable Ecosystem
```

Comment:
Would you consider a shorter, more direct title, such as "Practices for Productive Software Development in Kokkos"?  I understand the desire to emphasize Kokkos as a performance portability tool, but we're not really talking about the performance portability aspect of Kokkos (except in that it relates to the breadth of testing).  It is really about the development practices used within the Kokkos project.

Reply:
Good point and thanks for revision suggestion, addressed by 1a4586007f398b110bd1798dfb1e3ba12325aa18